pkg/testutil/fake/openstack/instance_client.go

```diff
@@ -1,11 +1,11 @@
 /**
- * Copyright 2019, Z Lab Corporation. All rights reserved.
+ * Copyright 2021, Z Lab Corporation. All rights reserved.
```

Comment:
Nice Catch!
Could you update the copyright year in all files as well as this change?

Reply:
I don't think it's necessary to update the year in copyright header of all files.
I intend I created this file as a new file, so I update it to the current year.
However if we regard this is only moving files, we might not need to change it in the first place.

Reply:
> I don't think it's necessary to update the year in copyright header of all files.

Exactly. 

> However if we regard this is only moving files, we might not need to change it in the first place.

It is fine as it is.